### PR TITLE
 Fix the version state of BlockVolume

### PR DESCRIPTION
**Pull request by houjun41544**

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -310,8 +310,9 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 ### Volume Mode
 
 {{< feature-state for_k8s_version="v1.9" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.13" state="beta" >}}
 
-To enable this feature, enable the `BlockVolume` feature gate on the apiserver, controller-manager and the kubelet.
+Surport for raw block volumes is now enabled by default.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Please remove line 312 and line 315

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -549,9 +550,9 @@ spec:
 ## Raw Block Volume Support
 
 {{< feature-state for_k8s_version="v1.9" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.13" state="beta" >}}
 
-To enable support for raw block volumes, enable the `BlockVolume` feature gate on the
-apiserver, controller-manager and the kubelet.
+Surport for raw block volumes is now enabled by default.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md]: Please remove line 552 and 555

**Comment by houjun41544** [comment]
@tengqm ok, These lines have been removed.

**Pull request merged**